### PR TITLE
feat: update xapian-core dependency of doxygen to most recent version

### PR DESCRIPTION
**Pull request by Pro**
Specify library name and version:  **doxygen/1.9.4**

xapian-core package is available as version 1.4.19.
Doxygen should use this most recent version.

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by Pro** [comment]
@uilianries seems the `continuous-integration/jenkins/pr-merge` job is stuck.
Rest is all green. Could you quickly check why?

**Comment by uilianries** [comment]
@Pro This PR should be merged automatically soon. Thank you for your update!

**Review by uilianries — APPROVED** [review]
LGTM

**Comment by uilianries** [comment]
@Pro This PR is no longer marked as bump-<something>, which means, it needs 2 positive reviews to be merged.

File: recipes/doxygen/all/conanfile.py [hunk 1]
```diff
@@ -1,5 +1,8 @@
-from conans import CMake, ConanFile, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conan.tools.scm import Version
+from conan import tools
+from conans import CMake
+from conan.errors import ConanInvalidConfiguration
 import os
 
 required_conan_version = ">=1.33.0"
```
Comment (prince-chrismc) [inline recipes/doxygen/all/conanfile.py:8]:
```suggestion
required_conan_version = ">=1.50.2"
```

**Review by prince-chrismc — COMMENTED** [review]
one minor comment

**Review by uilianries — APPROVED** [review]

**Pull request merged**